### DISCUSSION TITLE
Connection timed out when checking for updates

### DISCUSSION THREAD

**Discussion by HonokaKousaka** [Monika-After-Story/MonikaModDev#10443] [QA - tech support + general] [answered]
I really love this mod and I have played this mod for over 5 years, but recently I found that each time I wanted to check for updates, the game would tell me Connection timed out. Although I have the ability to update on my own, I am still looking forward to having an automatic updating tool. 
Does anyone know what I can do to deal with this problem? Is it actually normal? I found that on the official website of MAS, no new updates were released, so I was really worried about this mod.
Thank you taking time to read my issue. I hope someone could help me.

**Comment by HonokaKousaka** [comment]
Oh, I was wrong. The update on the official website is the latest version. 
But I am still curious on the Connection timed out problem. Could anyone tell me?

**Comment by JAKill02** [comment]
> Oh, I was wrong. The update on the official website is the latest version. But I am still curious on the Connection timed out problem. Could anyone tell me?

disable **Goodbye DPI**, or another program related to bypassing blocking on the Internet

**Comment by HonokaKousaka** [comment]
Hello, I am really glad that you replied to me! However, I have tried checking updates with and without my VPN turned on, but neither of them worked. I am not using Goodbye DPI. I had this problem actually one or several years ago, and I still do not know what to do. Sorry for the disruption again.

**Comment by Booplicate** [comment]
Does it always happen or just sometimes?

**Comment by HonokaKousaka** [comment]
I think it has lasted for 1 or 2 years? Each time I wanted to check the updates, Connection timed out would just appear and I have tried different methods but I still do not know how to solve it. :(

**Comment by Booplicate** [comment]
@ThePotatoGuy do you happen to know if the server is blocking connections?

**Comment by auto-ash** [comment]
I have this problem too,I saw the update message (like"a new version available" in the top right corner(I don't remember the accurate message)).So I went to update the game.However this problem occurred.This problem has existed may be 2 years?I used to think that it is because there has not been any new vision,so I didn't think it is a bug or something strange.

**Comment by HonokaKousaka** [comment, marked as answer]
Hello everyone! Several days ago, I found that this bug in my game had disappeared. I did not know the details and did not change any code, but I succeeded in updating the game through the update button in the game.
